src/registrar/models/user.py

```diff
@@ -163,6 +163,16 @@ def get_active_requests_count(self):
         active_requests_count = self.domain_requests_created.filter(status__in=allowed_states).count()
         return active_requests_count
 
+    def get_num_expiring_domains(self, request):
+        """Return number of expiring domains"""
+        domain_ids = self.get_user_domain_ids(request)
+        domains = Domain.objects.filter(id__in=domain_ids)
+        how_many_expired_domains = 0
```

Comment:
Let's rename this variable to num_expiring_domains since we're counting domains expiring and not the ones already expired